scania-digital-design-system/sdds
# fix(dropdown): standardise order of controls in Storybook

**Describe pull-request**  
Standardised order of controls in Storybook for dropdown component. Also updated descriptions, aligned default values to props and refactored deprecated controls, helper control and error state control.

**Solving issue**  
Fixes: [DTS-860](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-860)

**How to test**  
1. Go to Storybook link below
2. Check in Dropdown -> Native, Web Component Default and Web Component Multiselect
3. Check in Dropdown Filter -> Default
4. Inspect order of controls
5. Read control descriptions
6. Check that "Helper text" control works on all components
7. Check that "Error state" control works on all components

**Suggested test steps**

[DTS-860]: https://tegel.atlassian.net/browse/DTS-860?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
